recipe/meta.yaml

```diff
@@ -10,24 +10,22 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: c93fc0ed8216c71f9b8264788ca7743e26f4498b2011e5e2404696e5fd238e68
+  sha256: c8e9d769a6066bb5f94bb0362dbc20ed6f6b525336373263aeb6f9c6dc7b91e4
 
 build:
   number: 0
   skip: true  # [py<37 or osx]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
we can do this without using the scripts
```
script:
  - export CUDA_PATH=$PREFIX  # [linux]
  - set CUDA_PATH=%PREFIX%  # [win]
  - {{ PYTHON }} -m pip install . --no-deps -vv

Reply:
True, my concern there was it might start to look messy. I'll make the change